edagames/settings/prod.py

```diff
@@ -128,7 +128,7 @@ def get_secret(db_secret_name):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': secret_value["dbClusterIdentifier"],
+        'NAME': secret_value["dbname"],
```

Comment:
Why is this change necessary?

Reply:
Because the `dbClusterIdentifier` key of the secret value of AWS Secrets Manager is the name of the cluster, in our case,`aurora-edagames`. And Django needs the name of the database, not the cluster. So, I have to add a new key in the secret, called `dbname` with the name of the database as its value.